### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MichaelCurrin**
Improve grammar and flow of text 

Improve readability of example 

Bump required Go version to match https://github.com/cristalhq/aconfig/blob/main/aconfigyaml/go.mod

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
 # aconfig
+> Simple, useful and opinionated config loader
```
Comment (cristaloleg) [inline README.md:2]: why?

Comment (MichaelCurrin) [reply README.md:2]: The most common style on GitHub I've seen is title, description,  badges, and then other stuff.

I'll revert though

**Comment by cristaloleg** [comment]
Looks like most of the changes are stylistic rather than practical. I appreciate for your time but I'm not ready to merge it. Thank you.

**Comment by MichaelCurrin** [comment]
Fair enough. I reverted most of it and just left the grammar fix which is why I started the PR originally.

**Comment by cristaloleg** [comment]
Thank you!

**Pull request merged**